testnet/stacks-node/src/tests/neon_integrations.rs

```diff
@@ -1117,6 +1139,32 @@ fn bitcoind_forking_test() {
     channel.stop_chains_coordinator();
 }
 
+/// Returns a StacksMicroblock with the given transactions, sequence, and parent block that is
+/// signed with the given private key.
+fn make_signed_microblock(
+    block_privk: &StacksPrivateKey,
+    txs: Vec<StacksTransaction>,
+    parent_block: BlockHeaderHash,
+    seq: u16,
+) -> StacksMicroblock {
+    let txid_vecs = txs.iter().map(|tx| tx.txid().as_bytes().to_vec()).collect();
+    let merkle_tree = MerkleTree::<Sha512Trunc256Sum>::new(&txid_vecs);
+    let tx_merkle_root = merkle_tree.root();
+
+    let mut mblock = StacksMicroblock {
+        header: StacksMicroblockHeader {
+            version: 0x12,
```

Comment:
Just for fun, can you make `.version` a random value each time this test runs? I just want to make sure that the version field can be arbitrary -- i.e. it's not consensus-critical.